### PR TITLE
Syncer:  Refactors KCPClusterName to SyncTargetWorkspace.

### PR DESCRIPTION
**Pull request by jmprusi**
Various refactors and improvements to logging messages

### CODE REVIEW

File: cmd/syncer/cmd/syncer.go [hunk 1]
```diff
@@ -104,11 +104,11 @@ func Run(options *synceroptions.Options, ctx context.Context) error {
 	if err := syncer.StartSyncer(
 		ctx,
 		&syncer.SyncerConfig{
-			UpstreamConfig:   kcpConfig,
-			DownstreamConfig: toConfig,
-			ResourcesToSync:  sets.NewString(options.SyncedResourceTypes...),
-			KCPClusterName:   logicalcluster.New(options.FromClusterName),
-			SyncTargetName:   options.PclusterID,
+			UpstreamConfig:      kcpConfig,
```
Comment (sttts) [inline cmd/syncer/cmd/syncer.go:107]: upstreamConfig

File: cmd/syncer/cmd/syncer.go [hunk 2]
```diff
@@ -104,11 +104,11 @@ func Run(options *synceroptions.Options, ctx context.Context) error {
...
+			DownstreamConfig:    toConfig,
+			ResourcesToSync:     sets.NewString(options.SyncedResourceTypes...),
+			SyncTargetWorkspace: logicalcluster.New(options.FromClusterName),
+			SyncTargetName:      options.PclusterID,
```
Comment (sttts) [inline cmd/syncer/cmd/syncer.go:111]: how about renaming PclusterID to syncTargetName ?

File: cmd/syncer/cmd/syncer.go [hunk 3]
```diff
@@ -104,11 +104,11 @@ func Run(options *synceroptions.Options, ctx context.Context) error {
 	if err := syncer.StartSyncer(
 		ctx,
 		&syncer.SyncerConfig{
-			UpstreamConfig:   kcpConfig,
-			DownstreamConfig: toConfig,
-			ResourcesToSync:  sets.NewString(options.SyncedResourceTypes...),
-			KCPClusterName:   logicalcluster.New(options.FromClusterName),
-			SyncTargetName:   options.PclusterID,
+			UpstreamConfig:      kcpConfig,
+			DownstreamConfig:    toConfig,
```
[the remark this answers is not in the document]
Comment (jmprusi) [reply cmd/syncer/cmd/syncer.go]: done

File: pkg/syncer/spec/spec_controller.go [hunk 4]
```diff
@@ -157,7 +157,7 @@ func NewSpecSyncer(gvrs []schema.GroupVersionResource, syncTargetClusterName log
 				c.AddToQueue(gvr, m)
 			},
 		})
-		klog.V(2).InfoS("Set up downstream informer", "clusterName", syncTargetClusterName, "pcluster", syncTargetName, "gvr", gvr.String())
+		klog.V(2).InfoS("Set up downstream informer", "SyncTarget Workspace", syncTargetWorkspace, "SyncTarget Name", syncTargetName, "gvr", gvr.String())
```
[the remark this answers is not in the document]
Comment (jmprusi) [reply pkg/syncer/spec/spec_controller.go:160]: Should I remove the spaces?

File: pkg/syncer/spec/spec_controller.go [hunk 5]
```diff
@@ -58,12 +58,12 @@ type Controller struct {
 	upstreamInformers, downstreamInformers dynamicinformer.DynamicSharedInformerFactory
 
 	syncTargetName            string
-	syncTargetClusterName     logicalcluster.Name
+	syncTargetWorspace        logicalcluster.Name
```
Comment (stevekuznetsov) [inline pkg/syncer/spec/spec_controller.go:61]: worspace

**Comment by stevekuznetsov** [comment]
some typos, do we want to 

/hold
?

File: pkg/syncer/spec/spec_controller.go (hunk 5 above)
Comment (jmprusi) [reply pkg/syncer/spec/spec_controller.go:61]: uoops..

Comment (jmprusi) [reply pkg/syncer/spec/spec_controller.go]: fixed.

File: pkg/syncer/spec/spec_controller.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (jmprusi) [reply pkg/syncer/spec/spec_controller.go:160]: done, removed spaces, please retag!

File: pkg/syncer/spec/spec_controller.go [hunk 6]
```diff
@@ -111,7 +111,7 @@ func NewSpecSyncer(gvrs []schema.GroupVersionResource, syncTargetClusterName log
 				c.AddToQueue(gvr, obj)
 			},
 		})
-		klog.V(2).InfoS("Set up upstream informer", "clusterName", syncTargetClusterName, "pcluster", syncTargetName, "gvr", gvr.String())
+		klog.V(2).InfoS("Set up upstream informer", "syncTarget_workspace", syncTargetWorkspace, "synctarget_name", syncTargetName, "gvr", gvr.String())
```
Comment (sttts) [inline pkg/syncer/spec/spec_controller.go:114]: upper case or camlcase?

**Pull request merged**